### DISCUSSION TITLE
Help needed with pushing files from phpstorm to Github

### DISCUSSION THREAD

**Discussion by lukeMersh** [community/community#48803] [New to GitHub]
Dear friends
I have been following a youtube video on Wordpress development using Local by flywheel and Github.
My issue is that when i try to push files from phpstorm to github I get a permissions error.
Is there anybody that can help me with this issue.
kind regards

**Comment by SMelidoni** [comment]
Hi there @lukeMersh 👋🏽

Sorry to hear you're encountering error's while pushing files from PhpStorm to GitHub. To try and resolve this issue, please follow these steps:

1. **Check your SSH keys**: Make sure you've added your SSH key to your GitHub account. You can follow GitHub's [documentation on adding a new SSH key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) for guidance.

2. **Configure PhpStorm**: Ensure that PhpStorm is configured to use the correct SSH key for authentication. You can follow JetBrains' [documentation on configuring SSH keys in PhpStorm](https://www.jetbrains.com/help/phpstorm/create-ssh-configurations.html) for guidance.

3. **Verify remote URL**: Double-check the remote URL in your PhpStorm project settings and ensure it's using the SSH format (`git@github.com:user/repo.git`) instead of the HTTPS format (`https://github.com/user/repo.git`).

If you've completed the above steps and are still encountering the permissions error, please provide more details about the error message, and I'll be happy to assist you further!
